TacTok/options.py

```diff
@@ -9,18 +9,21 @@
 sys.path.append(os.path.sep.join(__file__.split(os.path.sep)[:-2]))
 from utils import log
 import pdb
-
+import pickle
 
 def parse_args():
     parser = argparse.ArgumentParser()
 
     parser.add_argument('--tac_grammar', type=str, default='tactics.ebnf')
     
+    # global options
+    parser.add_argument('--toggle_defs', action='store_false', help='toggle whether to include the names of definitions and theorems in the model')
+    parser.add_argument('--toggle_locals', action='store_false', help='toggle whether to include the names of local variables in the model')
 
     # experimental setup
     parser.add_argument('--include_synthetic', action='store_true')
     parser.add_argument('--exp_id', type=str)
-    parser.add_argument('--datapath', type=str, default='processed/proof_steps')
+    parser.add_argument('--datapath', type=str, default='proof_steps')
```

Comment:
Do I need to change this back, @efirst?

Reply:
I think so, mine are in `processed/proof_steps`